src/swagger.py

```diff
@@ -316,7 +316,7 @@ def process_sparql_query_text(query_text, loader, call_name, extraMetadata):
     elif query_metadata['type'] == 'ConstructQuery':
         if not method:
             method = 'get'
-    elif query_metadata['type'] == 'InsertData': # UPDATE queries should map here
+    elif query_metadata['type'] == 'InsertData' or query_metadata['type'] == 'Modify': # UPDATE queries should map here
```

Comment:
@c-martinez this indeed matches the *Modify* queries (such as the one discussed in https://github.com/CLARIAH/grlc/pull/400#issuecomment-1562283086).

Yet we must match the HTTP POST option to make the test successful.

Options here are:

```
    elif query_metadata['type'] == 'InsertData' or query_metadata['type'] == 'Modify': # UPDATE queries should map here
        if not method:
            method = 'post'
```

or (cumbersome):

```
    elif query_metadata['type'] == 'InsertData' or query_metadata['type'] == 'Modify': # UPDATE queries should map here
        pass
```

... otherwise we keep falling back to the trap (few lines below):
```
raise Exception('Could not parse query {}: Query of type {} is currently unsupported'.format(call_name, query_metadata['type']))
```